api/v1/routes/iterations.rb

```diff
@@ -14,7 +14,9 @@ class Iterations < Core
           halt 401, { error: message }.to_json
         end
 
-        unless Xapi.exists?(language, slug)
+        track = Trackler.tracks[language]
```

Comment:
Let's rename `language` to `track_id` for clarity (I know it was already `language` in the route definition, but it would be great to get this cleaned up as we refactor towards trackler).